modules/gitops-release-notes-1-9-1.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-9-1_{context}"]
+= Release notes for {gitops-title} 1.9.1
+
+{gitops-title} 1.9.1 is now available on {product-title} 4.12 and 4.13.
+
+[id="errata-updates-1-9-1_{context}"]
+== Errata updates
+
+=== RHSA-2023:3591- {gitops-title} 1.9.1 security update advisory 
+
+Issued: 2023-07-10
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:3591[RHSA-2023:3591]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="new-features-1-9-1_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the bundled Argo CD has been updated to version 2.7.6.
+
+[id="fixed-issues-1-9-1_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, some users were observing that Argo CD was becoming unresponsive with an increase in namespaces and applications. This update fixes the issue by removing a deadlock happening when two functions were competing for mutexes. Now, users should not experience crashes or unresponsiveness when they have a large number of resources. link:https://issues.redhat.com/browse/GITOPS-2782[GITOPS-2782]
+
+* Before this update, users reported that `argocd-application-controller` would suddenly stop working when re-syncing. This update fixes the issue by adding logic to prevent a cluster cache deadlock. Now, users should no longer face this deadlock situation and applications should resync successfully. link:https://issues.redhat.com/browse/GITOPS-2880[GITOPS-2880]
```

Comment:
Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting and carefully handle the following instances:

-  `argocd-application-controller`
- `known_host` (the succeeding RN text)
- `argocd-ssh-knownhosts-cm` (the succeeding RN text)
-  configmap
- appSync setting